docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -1615,3 +1605,190 @@ Suggested change to the schema is to remove this redundant index `idx_orders_ord
 ```sql
 CREATE INDEX idx_orders_order_id on orders(order_id);
 ```
+
+---
+
+### Index on low-cardinality column
+
+**Description**:
+
+In YugabyteDB, the columns specified in the [CREATE INDEX](../../../api/ysql/the-sql-language/statements/ddl_create_index) statement are of three kinds: sharding, clustering, and covering. For more details, refer to the documentation for [secondary indexes](../../../explore/ysql-language-features/indexes-constraints/secondary-indexes-ysql/). The sharding strategy is HASH by default unless [Postgres Compatibility mode](./../../develop/postgresql-compatibility/) is enabled, in which case, RANGE is the default sharding strategy.
+
+Design the index to evenly distribute data across all nodes and optimize performance based on query patterns. Avoid using low-cardinality columns, such as boolean values, ENUMs, or days of the week, as sharding keys, as they result in data being distributed across only a few tablets.
+
+#### Single column index
+
+Using a single-column index on a low-cardinality column leads to uneven data distribution, regardless of the sharding strategy.
+
+**Workaround**:
+
+It is recommended to drop the index if it is not required..
+
+If the index is used in queries, combine it with a high-cardinality column to create either a multi-column index with the sharding key on the high-cardinality column or a multi-column range-sharding index. This ensures better data distribution across all nodes.
+
+#### Multi-column index
+
+In a multi-column index with a low cardinality column as the sharding key, the data will be unevenly distributed.
+
+**Workaround**:
+
+Make this index a range-sharded index to distribute data based on the combined values of all columns, or reorder the index columns to place the high-cardinality column first. This enables sharding on the high-cardinality column and ensures even distribution across all nodes.
+
+**Example**:
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TYPE order_statuses AS ENUM ('CONFIRMED', 'SHIPPED', 'OUT FOR DELIVERY', 'DELIVERED', 'CANCELLED');
+
+CREATE TABLE orders (
+    order_id int PRIMARY,
+    ...,
+    status order_statuses
+);
+
+CREATE INDEX idx_order_status on orders (status); --single column index on column having only 5 values 
+
+CREATE INDEX idx_order_status_order_id on orders (status, order_id); --multi column index on first column with only 5 values 
+```
+
+Since the no. of distinct values of the column status is 5, there will be a maximum of 5 tablets created, limiting the scalability.
```

Comment:
```suggestion
Since the number of distinct values of the column `status` is 5, there will be a maximum of 5 tablets created, limiting the scalability.
```